pkg/webui/console/store/actions/lib.js

```diff
@@ -55,8 +55,27 @@ export const attachPromise = actionOrActionCreator => {
     return decorateAction(actionOrActionCreator)
   }
 
-  return function(...args) {
+  return (...args) => {
     const action = actionOrActionCreator(...args)
     return decorateAction(action)
   }
 }
+
+/**
+ * Helper function to retrieve the result action types based
+ * on the request action type.
+ *
+ * @param {object|string} typeString - The request action type.
+ * @param {string} status - The result type, either `SUCCESS` or `FAILURE`.
+ * @returns {string} - The result action type.
+ */
+export const getResultActionFromType = (typeString, status) => {
+  if (typeString instanceof Array) {
```

Comment:
I actually just moved this here from `/webui/console/store/middleware/logics/lib.js`. I don't remember why I included the possibility to use arrays too. I'll remove it since it does not seem to be used anywhere.